class.php

```diff
@@ -102,6 +103,11 @@ public function setCustomRoutes(array $customRoutes)
     $this->_customRoutes = $customRoutes;
   }
 
+  public function setIgnoreUntranslatedPages(bool $ignoreUntranslatedPages = null)
```

Comment:
```suggestion
  public function setIgnoreUntranslatedPages(bool $ignoreUntranslatedPages)
```
No need to make the argument optional. It's either called not at all or manually (and then there will be an argument) or by the plugin itself with `false`